packages/application-shell/src/components/navbar/navbar.js

```diff
@@ -189,33 +187,14 @@ export class MenuItemLink extends React.PureComponent {
   static propTypes = {
     linkTo: PropTypes.string,
     exactMatch: PropTypes.bool,
-    externalLink: PropTypes.string,
-    tracking: PropTypes.shape({
-      'data-track-component': PropTypes.string,
-      'data-track-event': PropTypes.string,
-      'data-track-label': PropTypes.string,
-    }),
     children: PropTypes.node.isRequired,
     useFullRedirectsForLinks: PropTypes.bool.isRequired,
   };
   static defaultProps = {
     exactMatch: false,
-    tracking: {},
   };
   redirectTo = targetUrl => window.location.replace(targetUrl);
   render() {
-    if (this.props.externalLink) {
```

Comment:
This case was only used by the support link, which is now rendered separately. Hence the cleanup here